wazuh/wazuh-splunk
# Fix non-sortable columns on the Security section tables

Summary
----------

This PR fixes sorting issues on the Security section tables, as some columns were able to be sorted when it shouldn't.

To test
-----

Simply go to the Security sections and check the sorting on the table. Check that no error occurs.

The following columns must not be sortable:
* Users : run_as, roles
* Roles : policies
* Role Mapping : roles
